src/kirby/components/checkbox/checkbox.component.spec.ts

```diff
@@ -0,0 +1,42 @@
+/* tslint:disable:no-unused-variable */
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { MockComponent } from 'ng-mocks';
+import * as ionic from '@ionic/angular';
+import { By } from '@angular/platform-browser';
+import { DebugElement } from '@angular/core';
+
+import { CheckboxComponent } from './checkbox.component';
+
+describe('CheckboxComponent', () => {
```

Comment:
Consider testing the logic in the template "switching" mode. You can find documentation how to test the DOM here [Angular - testing](https://angular.io/guide/testing#component-dom-testing)

Reply:
As an addition to @hafstad's comment, you should also test that all properties are bound in the template.

Reply:
Could you elaborate on said "bound properties?" :)